fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/RecordQueryPlanner.java

```diff
@@ -545,6 +546,15 @@ private ScoredPlan planIndex(@Nonnull PlanContext planContext, @Nonnull QueryCom
                     p = computeIndexFilters(planContext, p);
                 }
                 return p;
+            } else if (indexTypes.getLuceneTypes().contains(index.getType())) {
+                p = planLucene(candidateScan, index, filter, sort);
```

Comment:
Is this plan (and the index it uses) correctly communicating `createsDuplicates()`?

Reply:
Since we're assuming a flat index I'm fairly certain this should be being correctly used from what we discussed before right?